ui/components/institutional/custody-confirm-link-modal/custody-confirm-link-modal.js

```diff
@@ -48,9 +48,8 @@ const CustodyConfirmLink = ({ hideModal }) => {
   const { custodians } = useSelector(getMMIConfiguration);
   const { custodianName } =
     custodyAccountDetails[toChecksumHexAddress(address)] || {};
-  const { displayName, iconUrl } = custodians.find(
-    (item) => item.name === custodianName || {},
-  );
+  const { displayName, iconUrl } =
```

Comment:
In this updated version of the code, only a single argument is passed to the find method, which is the callback function used to test each element of the array. Additionally, a default value of an empty object {} is provided using the logical OR operator ||, in case no element in the array passes the test and the find method returns undefined.